### DISCUSSION TITLE
Always displaying the Jog, Goto, Outline, MPos etc

### DISCUSSION THREAD

**Discussion by humneraudio** [terjeio/ioSender#185] [Ideas]
Could it be an option to have Jog, Goto, Outline, MPos etc always viewable? I noticed a lot of empty space under DRO, could the MPos be displayed under this, I always reference both WPOS and MPOS when I position my jobs etc? And this goes for the other tabs too, Goto and Outline I think should move out to these positions.

**Comment by terjeio** [comment]
Check the latest [XL Edge version](http://www.io-engineering.com/downloads).

>  I noticed a lot of empty space under DRO, could the MPos be displayed under this, I always reference both WPOS and MPOS when I position my jobs etc?

The sender supports up to 6 axes and the _DRO_ and _Program limits_ groups will grow for that - so not really any free space to use in a consistent manner. For MPOS visibility you can enable text overlay in the 3D viewer.

> And this goes for the other tabs too, Goto and Outline I think should move out to these positions.

The XL version has such a layout - but needs a slightly larger display resolution.
